src/com/sun/ts/tests/securityapi/idstore/ldap/invalidgroupnameattr/ServletForLdapIDStore.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2017, 2020 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
```

Comment:
The changes on this file do not look needed.

Reply:
Fixed.